TFproviderTest.yml

```diff
@@ -226,15 +227,18 @@ pipelines:
             - export PR_URL=${res_GitHubTFProviderXrayRepoJFrog_commitUrl}
             - export PR_COMMITTER=${res_GitHubTFProviderXrayRepoJFrog_committerLogin}
             - export PR_BRANCH=${res_GitHubTFProviderXrayRepoJFrog_headCommitRef}
-            - export PR_TITLE=${res_GitHubTFProviderXrayRepoJFrog_commitMessage}
+            - export PR_TITLE=$(echo ${res_GitHubTFProviderXrayRepoJFrog_commitMessage} | sed -e 's/\&/\&amp;/g' -e 's/>/\&gt;/g' -e 's/</\&lt;/g')
```

Comment:
Please add comment here on why we escaping three characters

Reply:
done